File: app/src/main/java/net/squanchy/navigation/BottomNavigationSection.java
package net.squanchy.navigation;

import android.support.annotation.StyleRes;

import net.squanchy.R;

public enum BottomNavigationSection {
    SCHEDULE(R.style.Theme_Squanchy_Schedule),
    FAVORITES(R.style.Theme_Squanchy_Favorites),
    TWEETS(R.style.Theme_Squanchy_Tweets),
    VENUE_INFO(R.style.Theme_Squanchy_VenueInfo);

    private final int theme;

    BottomNavigationSection(@StyleRes int theme) {
        this.theme = theme;
    }

    @StyleRes
    public int theme() {
        return theme;
    }
}

File: app/src/main/java/net/squanchy/home/BottomNavigationSection.kt
package net.squanchy.home

import androidx.annotation.StyleRes

import net.squanchy.R

enum class BottomNavigationSection(@param:StyleRes val theme: Int) {
    SCHEDULE(R.style.Theme_Squanchy_Schedule),
    FAVORITES(R.style.Theme_Squanchy_Favorites),
    TWEETS(R.style.Theme_Squanchy_Tweets),
    VENUE_INFO(R.style.Theme_Squanchy_VenueInfo);
}
